chroma-manager/chroma_core/management/commands/print-settings.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2018 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a MIT-style
+# license that can be found in the LICENSE file.
+
+import settings
+import glob
+import json
+from django.db import connection
+from django.core.management.commands.runserver import Command as BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Prints out selected settings in env variable format
+    """
+    help = """Prints out selected settings in env variable format"""
+
+    def handle(self, *args, **kwargs):
+        cursor = connection.cursor()
+        cursor.execute("SELECT * from api_key()")
+        (API_USER, API_KEY) = cursor.fetchone()
+        cursor.close()
+
+        source_map_paths = glob.glob(
+            '/usr/share/iml-manager/iml-gui/main*.map')
+        SOURCE_MAP_PATH = next(iter(source_map_paths), None)
+
+        xs = map(lambda (x, y): "{0}={1}".format(x, y), [
```

Comment:
why don't we just expose every setting in the settings file and remove the ones we don't need? then we don't have to have a separate list

Reply:
Because not everything has a direct representation to an environment file, and we'd need to enumerate everything to properly serialize the differences.